### PR TITLE
Update lock files for changes made in 57517f9 and 705514c

### PR DESCRIPTION
**Pull request by jasonmalinowski**
*Review*: @dotnet/roslyn-interactive, @dotnet/roslyn-infrastructure, @tmat

### CODE REVIEW

File: src/Interactive/VbiCore/project.lock.json [hunk 1]
```diff
@@ -1175,33 +1175,6 @@
           "ref/dotnet5.4/Microsoft.Win32.Primitives.dll": {}
         }
       },
-      "runtime.any.System.Linq.Expressions/4.0.11-beta-23504": {
```
Comment (amcasey) [inline src/Interactive/VbiCore/project.lock.json]: It looks like the version numbers got smaller.  Expected?

**Comment by amcasey** [comment]
You have met the extremely low bar of not having done anything I know to be incorrect. :)

File: src/Interactive/VbiCore/project.lock.json (hunk 1 above)
Comment (jasonmalinowski) [inline src/Interactive/VbiCore/project.lock.json]: The current version stated is this:

    src/Scripting/Core/project.json:14:    "System.Linq.Expressions": "4.0.10"`

So something else must have had a dependency pulling it forward that is no longer doing so.

**Comment by agocke** [comment]
I've been working on unifying and simplifying these refs in my `features/coreclr` branch, but this looks :+1: for now.

**Pull request merged**